src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsEvolveOasis.java

```diff
@@ -8,42 +8,42 @@
 
 import org.testng.annotations.Test;
 
-public class TestEvolveAds extends TemplateNoFirstLoad {
+public class TestAdsEvolveOasis extends TemplateNoFirstLoad {
 
   @GeoEdgeBrowserMobProxy(country = "CA")
   @UseUnstablePageLoadStrategy
   @Test(
       dataProviderClass = AdsDataProvider.class,
-      groups = {"TestEvolveAds"},
+      groups = "AdsEvolveOasis",
       dataProvider = "evolveTestPage"
   )
-  public void testEvolveCall_CA(String wikiName, String article) {
-    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
-    AdsEvolveObject wikiPage = new AdsEvolveObject(driver, testedPage);
-    wikiPage.verifyEvolveCall();
+  public void adsEvolveOasisCA(String wikiName, String article) {
+    adsEvolveOasis(wikiName, article);
   }
 
   @GeoEdgeBrowserMobProxy(country = "AU")
```

Comment:
O there is this library with CountryCode enum but this library would be an overkill in my opinion.
http://stackoverflow.com/questions/139867/is-there-an-open-source-java-enum-of-iso-3166-1-country-codes